src/server/body-parser.ts

```diff
@@ -42,6 +42,11 @@ export let bodyParser = {
    * }
    */
   async toJSON(request: Request): Promise<unknown> {
+    if (request.headers.get("Content-Type") === "application/json") {
+      let rawBody = await this.toString(request);
```

Comment:
This could be request.json() directly if you know the content type.

Reply:
updated, thanks for the tips. looks way cleaner now 👌